UPGRADING.md

```diff
@@ -1,6 +1,31 @@
 Upgrading Grape
 ===============
 
+
+### Upgrading to >= 1.5.3
+
+### Nil value and coercion
+
+A parameter with a value set to `nil` call custom coercion method, but the coercion is not triggered for missing parameters.
```

Comment:
Let's find when the wrong behavior was introduced, try against 1.4.0 and up and let's make sure we are clear how it behaved and that we're _fixing_ this now. I think the text should say something like this:

Prior to version 1.x.0 passing a `nil` value for a parameter with a customer coercer would invoke the coercer, and not passing a value would not invoke it. This behavior was not tested or documented. Version 1.z.0 quietly changed this behavior, in such that ... . Version 1.5.3 fixes and documents this as follows: .... 

Makes sense?

Reply:
mm, I think this behavior was introduced in the migration to `dry-types`, see https://github.com/ruby-grape/grape/commit/9b352a88ff3a4970af7bfa3c633c09e956c2e8ea#diff-6158291f8f15938bb4d6460b274b5106132113841ede636d29d20ef3d01f21b7R53

so it should be reproducible starting from 1.3.0

Reply:
@dm1try thanks for the hint. I confirm that the skip of `nil` value was introduced in v1.3.0, and version v1.2.5 met the expected behavior.